tools/benchmark_tool/openvino/tools/benchmark/main.py

```diff
@@ -170,7 +166,7 @@ def set_throughput_streams():
                     else:
                         raise Exception(f"Device {device} doesn't support config key '{key}'! " +
                                         "Please specify -nstreams for correct devices in format  <dev1>:<nstreams1>,<dev2>:<nstreams2>")
-                elif key not in config[device].keys() and args.api_type == "async" and not is_flag_set_in_command_line('hint'):
+                elif key not in config[device].keys() and args.api_type == "async" and config[device]['PERFORMANCE_HINT'] == "":
```

Comment:
Maybe check that `PERFORMANCE_HINT` is in map actually?

Reply:
The reason for this check is that in line 88-90 of main.py the execution for hint == 'none' was as follows:
` if args.perf_hint=='none':
            logger.warning(f"No device {device} performance hint is set.")
            args.perf_hint = ""`
And then it was assigned to config[device] in lines 147-148:
`if is_flag_set_in_command_line('hint') or args.perf_hint:
                config[device]['PERFORMANCE_HINT'] = args.perf_hint.upper()`

I removed the line 90, so that perf_hint stays 'none', and added the following check:
`if not args.perf_hint == "none":
             config[device]['PERFORMANCE_HINT'] = args.perf_hint.upper()`
This now allows to check for 'PERFORMANCE_HINT' not in config[device].keys().
I verified that both before and after changes, the HINT states 'UNDEFINED' in the device configuration.
@nkogteva can you verify whether everything is correct now? :)

Reply:
@PiotrKrzem My point was that we need to check
`'PERFORMANCE_HINT' in config[device].keys() and config[device]['PERFORMANCE_HINT'] == ""` instead of just `config[device]['PERFORMANCE_HINT'] == ""`

But we difenetly shoud set 'PERFORMANCE_HINT' when hint is none, otherwise OV would not get this information (please rollback chanes on line 90)

Reply:
Okay, I guess I overcomplicated this unneccesarily. Rolled back